### PR TITLE
python3Packages.websocket_client: 0.57.0 -> 0.58.0, python3Packages.devolo-home-control-api: 0.16.0 -> 0.17.0

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream releases

Change log:

- https://github.com/websocket-client/websocket-client/blob/master/ChangeLog
- https://github.com/2Fake/devolo_home_control_api/releases/tag/v0.17.0

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 116265` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
8 packages marked as broken and skipped:
</summary>
<ul>
<li>awsebcli</li>
<li>mlflow-server</li>
<li>python38Packages.flower</li>
<li>python38Packages.mlflow</li>
<li>python38Packages.rl-coach</li>
<li>python39Packages.flower</li>
<li>python39Packages.mlflow</li>
<li>python39Packages.rl-coach</li>
</ul>
</details>
<details>

  <summary>1 package failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>python38Packages.gradient: log was empty</li>
  </ul>
</details>
<details>

  <summary>1 package failed to build and are new build failure:</summary>
  <ul>
<li>appdaemon: <a href="https://termbin.com/cour">plain log</a>  |  <a href="https://htmlpreview.github.io/?https://gist.githubusercontent.com/SuperSandroBot/f088ac96f02f7a70d9e7bb17a4583cf2/raw/index.html">pretty log </a></li>
  </ul>
</details>
<details>
<summary>
94 packages built:
</summary>
<ul>
<li>arion</li>
<li>aws-sam-cli</li>
<li>azure-cli</li>
<li>buildbot (python38Packages.buildbot)</li>
<li>buildbot-full (python38Packages.buildbot-full)</li>
<li>buildbot-ui (python38Packages.buildbot-ui)</li>
<li>dd-agent</li>
<li>docker-compose</li>
<li>errbot</li>
<li>fdroidserver</li>
<li>jellyfin-mpv-shim</li>
<li>ledger_agent (python38Packages.ledger_agent)</li>
<li>mpfshell</li>
<li>octoprint</li>
<li>onionshare</li>
<li>onionshare-gui</li>
<li>patroni</li>
<li>plex-mpv-shim</li>
<li>python38Packages.aioambient</li>
<li>python38Packages.aria2p</li>
<li>python38Packages.celery</li>
<li>python38Packages.devolo-home-control-api</li>
<li>python38Packages.django-raster</li>
<li>python38Packages.djmail</li>
<li>python38Packages.docker</li>
<li>python38Packages.docker-py</li>
<li>python38Packages.dockerspawner</li>
<li>python38Packages.flask-socketio</li>
<li>python38Packages.jellyfin-apiclient-python</li>
<li>python38Packages.jupyter-repo2docker</li>
<li>python38Packages.kubernetes</li>
<li>python38Packages.ledgerblue</li>
<li>python38Packages.libsoundtouch</li>
<li>python38Packages.moto</li>
<li>python38Packages.plexapi</li>
<li>python38Packages.praw</li>
<li>python38Packages.pushbullet</li>
<li>python38Packages.python-engineio</li>
<li>python38Packages.python-engineio_3</li>
<li>python38Packages.python-mpv-jsonipc</li>
<li>python38Packages.python-socketio</li>
<li>python38Packages.python-socketio_4</li>
<li>python38Packages.samsungctl</li>
<li>python38Packages.samsungtvws</li>
<li>python38Packages.slack-sdk</li>
<li>python38Packages.slackclient</li>
<li>python38Packages.sopel</li>
<li>python38Packages.tensorboardx</li>
<li>python38Packages.waterfurnace</li>
<li>python38Packages.websocket_client</li>
<li>python39Packages.aioambient</li>
<li>python39Packages.celery</li>
<li>python39Packages.devolo-home-control-api</li>
<li>python39Packages.django-raster</li>
<li>python39Packages.djmail</li>
<li>python39Packages.docker</li>
<li>python39Packages.docker-py</li>
<li>python39Packages.dockerspawner</li>
<li>python39Packages.flask-socketio</li>
<li>python39Packages.jellyfin-apiclient-python</li>
<li>python39Packages.jupyter-repo2docker</li>
<li>python39Packages.kubernetes</li>
<li>python39Packages.ledgerblue</li>
<li>python39Packages.libsoundtouch</li>
<li>python39Packages.moto</li>
<li>python39Packages.praw</li>
<li>python39Packages.pushbullet</li>
<li>python39Packages.python-engineio</li>
<li>python39Packages.python-engineio_3</li>
<li>python39Packages.python-socketio</li>
<li>python39Packages.python-socketio_4</li>
<li>python39Packages.samsungctl</li>
<li>python39Packages.samsungtvws</li>
<li>python39Packages.slack-sdk</li>
<li>python39Packages.slackclient</li>
<li>python39Packages.sopel</li>
<li>python39Packages.tensorboardx</li>
<li>python39Packages.waterfurnace</li>
<li>python39Packages.websocket_client</li>
<li>sourcehut.buildsrht</li>
<li>sourcehut.dispatchsrht</li>
<li>sourcehut.gitsrht</li>
<li>sourcehut.hgsrht</li>
<li>sourcehut.hubsrht</li>
<li>sourcehut.listssrht</li>
<li>sourcehut.mansrht</li>
<li>sourcehut.metasrht</li>
<li>sourcehut.pastesrht</li>
<li>sourcehut.todosrht</li>
<li>streamlink</li>
<li>streamlink-twitch-gui-bin</li>
<li>ulauncher</li>
<li>weechatScripts.wee-slack</li>
<li>zeronet</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
@fabaff can you patch the version requirement in appdaemon?

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 116265` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages marked as broken and skipped:</summary>
  <ul>
    <li>awsebcli</li>
    <li>mlflow-server</li>
    <li>python38Packages.flower</li>
    <li>python38Packages.mlflow</li>
    <li>python38Packages.rl-coach</li>
    <li>python39Packages.flower</li>
    <li>python39Packages.mlflow</li>
    <li>python39Packages.rl-coach</li>
  </ul>
</details>
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>python38Packages.gradient</li>
  </ul>
</details>
<details>
  <summary>96 packages built:</summary>
  <ul>
    <li>appdaemon</li>
    <li>arion</li>
    <li>aws-sam-cli</li>
    <li>azure-cli</li>
    <li>buildbot (python38Packages.buildbot)</li>
    <li>buildbot-full (python38Packages.buildbot-full)</li>
    <li>buildbot-ui (python38Packages.buildbot-ui)</li>
    <li>dd-agent</li>
    <li>docker-compose</li>
    <li>errbot</li>
    <li>fdroidserver</li>
    <li>jellyfin-mpv-shim</li>
    <li>ledger_agent (python38Packages.ledger_agent)</li>
    <li>mpfshell</li>
    <li>octoprint</li>
    <li>onionshare</li>
    <li>onionshare-gui</li>
    <li>patroni</li>
    <li>plex-mpv-shim</li>
    <li>python38Packages.aioambient</li>
    <li>python38Packages.aria2p</li>
    <li>python38Packages.celery</li>
    <li>python38Packages.devolo-home-control-api</li>
    <li>python38Packages.django-raster</li>
    <li>python38Packages.djmail</li>
    <li>python38Packages.docker</li>
    <li>python38Packages.docker-py</li>
    <li>python38Packages.dockerspawner</li>
    <li>python38Packages.flask-socketio</li>
    <li>python38Packages.jellyfin-apiclient-python</li>
    <li>python38Packages.jupyter-repo2docker</li>
    <li>python38Packages.kubernetes</li>
    <li>python38Packages.ledgerblue</li>
    <li>python38Packages.libsoundtouch</li>
    <li>python38Packages.moto</li>
    <li>python38Packages.plexapi</li>
    <li>python38Packages.praw</li>
    <li>python38Packages.pushbullet</li>
    <li>python38Packages.python-engineio</li>
    <li>python38Packages.python-engineio_3</li>
    <li>python38Packages.python-mpv-jsonipc</li>
    <li>python38Packages.python-socketio</li>
    <li>python38Packages.python-socketio_4</li>
    <li>python38Packages.samsungctl</li>
    <li>python38Packages.samsungtvws</li>
    <li>python38Packages.slack-sdk</li>
    <li>python38Packages.slackclient</li>
    <li>python38Packages.sopel</li>
    <li>python38Packages.tensorboardx</li>
    <li>python38Packages.waterfurnace</li>
    <li>python38Packages.websocket_client</li>
    <li>python39Packages.aioambient</li>
    <li>python39Packages.celery</li>
    <li>python39Packages.devolo-home-control-api</li>
    <li>python39Packages.django-raster</li>
    <li>python39Packages.djmail</li>
    <li>python39Packages.docker</li>
    <li>python39Packages.docker-py</li>
    <li>python39Packages.dockerspawner</li>
    <li>python39Packages.flask-socketio</li>
    <li>python39Packages.jellyfin-apiclient-python</li>
    <li>python39Packages.jupyter-repo2docker</li>
    <li>python39Packages.kubernetes</li>
    <li>python39Packages.ledgerblue</li>
    <li>python39Packages.libsoundtouch</li>
    <li>python39Packages.moto</li>
    <li>python39Packages.praw</li>
    <li>python39Packages.pushbullet</li>
    <li>python39Packages.python-engineio</li>
    <li>python39Packages.python-engineio_3</li>
    <li>python39Packages.python-socketio</li>
    <li>python39Packages.python-socketio_4</li>
    <li>python39Packages.samsungctl</li>
    <li>python39Packages.samsungtvws</li>
    <li>python39Packages.slack-sdk</li>
    <li>python39Packages.slackclient</li>
    <li>python39Packages.sopel</li>
    <li>python39Packages.tensorboardx</li>
    <li>python39Packages.waterfurnace</li>
    <li>python39Packages.websocket_client</li>
    <li>sourcehut.buildsrht</li>
    <li>sourcehut.dispatchsrht</li>
    <li>sourcehut.gitsrht</li>
    <li>sourcehut.hgsrht</li>
    <li>sourcehut.hubsrht</li>
    <li>sourcehut.listssrht</li>
    <li>sourcehut.mansrht</li>
    <li>sourcehut.metasrht</li>
    <li>sourcehut.pastesrht</li>
    <li>sourcehut.todosrht</li>
    <li>streamlink</li>
    <li>streamlink-twitch-gui-bin</li>
    <li>tts</li>
    <li>ulauncher</li>
    <li>weechatScripts.wee-slack</li>
    <li>zeronet</li>
  </ul>
</details>
